src/allowedTermsBehind.js

```diff
@@ -0,0 +1,22 @@
+import actions from './types/court-action-types'
```

Comment:
Maybe this might be better placed in an `actions/` or other directory. For a new person to this repo, this file does not seem immediately obvious what its purpose is for :).

A couple comments explaining it might also be useful!